src/counts.jl

```diff
@@ -199,6 +231,24 @@ end
 
 countmap{T}(x::AbstractArray{T}) = addcounts!(Dict{T,Int}(), x)
 countmap{T,W}(x::AbstractArray{T}, wv::WeightVec{W}) = addcounts!(Dict{T,W}(), x, wv)
+"""
+    countmap(x::AbstractArray{T}, wv::WeightVec{W})
+
+### Args:
+* `x`: An `AbstractArray`.
```

Comment:
Actually, this probably means that giving a list of arguments isn't really useful. In most cases, you already mention them in the description of the function. I think you'd better ensure you mention all of them in that description, and get rid of the arguments list altogether.